Packages/Tracking/Core/Runtime/Plugins/LeapCSharp/Connection.cs

```diff
@@ -1024,6 +1024,56 @@ static public eLeapPolicyFlag FlagForPolicy(Controller.PolicyFlag singlePolicy)
             }
         }
 
+        /// <summary>
+        /// Temporarily makes a connection to determine if a Service is available.
+        /// Returns the result and closes the temporary connection upon completion.
+        /// </summary>
+        public static bool IsConnectionAvailable(string serverNamespace = "Leap Service")
```

Comment:
There was a time where creating short lived connections was something we weren't keen on doing. I assume we're good to do this now?

Reply:
I chatted with Tom Hey, he says this is fine and there is a _better_ way to get the service status coming soon, but this older approach will still need to remain for older version of the Service

Reply:
Yes, I had that change in mind when writing these comments. If Tom's happy then that's great.